src/Events/Services/Interfaces/IEventsService.cs

```diff
@@ -20,22 +20,19 @@ public interface IEventsService
         Task<string> Save(CloudEvent cloudEvent);
 
         /// <summary>
-        /// Push cloud event to inbound queue.
-        /// Should only be attempted after saving to persistent storage. 
+        /// Push cloud event to registration queue.
         /// </summary>
         /// <param name="cloudEvent">The cloudEvent to be queued</param>
         /// <returns>Id for the queued event</returns>
-        Task<string> PostInbound(CloudEvent cloudEvent);
+        Task<string> PostRegistration(CloudEvent cloudEvent);
```

Comment:
Started a simillar implementation for #103.
I went for the name registerEvent for this method as is does more than simply post the content to a queue, but also assign some values. Any thoughts? 

Here's parts of the summary that I set up so far. 
	/// <summary>
        /// Starts the process of registering an event 
        /// </summary>
        /// <param name="couldEvent">The cloud events to be registered</param>

Reply:
I agree in principle. What about RegisterNew(cloudEvent) ?